zhaquirks/bosch/isw-zdl1-wp11g.py

```diff
@@ -0,0 +1,71 @@
+"""Device handler for Bosch ISWZDL1WP11G motion sensor."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import Basic, Identify, Ota, PollControl
+from zigpy.zcl.clusters.homeautomation import Diagnostic
+from zigpy.zcl.clusters.measurement import TemperatureMeasurement
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks import PowerConfigurationCluster
+
+from . import BOSCH
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class BoschWP11GPowerConfiguration(PowerConfigurationCluster):
+    """Bosch power configuration cluster for ISWZDL1WP11G motion sensor."""
+
+    MAX_VOLTS = 6.0
+    MIN_VOLTS = 3.0
+
+
+class ISWZDL1WP11G(CustomDevice):
+    """Custom device representing Bosch ISWZDL1WP11G motion sensor."""
+
+    signature = {
+        #  <SimpleDescriptor endpoint=1 profile=260 device_type=1026
```

Comment:
The endpoint id in the signature and here in the comment do not match. Which one is right?

Reply:
verified in Home Assistant it is 5 and working properly as configured.  will correct comment.  Thank you.